bindings/python/pymongoarrow/lib.pyx

```diff
@@ -25,6 +25,7 @@ import enum
 import numpy as np
 from pyarrow import timestamp
 from pymongoarrow.types import _BsonArrowTypes
+from pymongoarrow.errors import *
```

Comment:
Can you change this to explicitly import what we need? It makes it easier to track what's part of the api.

Reply:
Done.